v3/src/components/tool-shelf/options-button.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react"
+import { Menu, MenuButton, MenuItem, MenuList, Tag, useDisclosure } from "@chakra-ui/react"
+import OptionsIcon from "../../assets/icons/icon-options.svg"
+import { useDocumentContext } from "../../hooks/use-document-context"
+import t from "../../utilities/translation/translate"
+import { kWebViewTileType } from "../web-view/web-view-defs"
+import { isWebViewModel } from "../web-view/web-view-model"
+import { WebViewUrlModal } from "../web-view/web-view-url-modal"
+
+import "./tool-shelf.scss"
+
+export const OptionsShelfButton = () => {
+  const document = useDocumentContext()
+  const formulaModal = useDisclosure()
```

Comment:
I don't understand "formula" in the name. "urlModal"?

Reply:
Good catch, that's left over from copying and pasting.